src/main/resources/regression/beir-v1.0.0-robust04-flat.yaml

```diff
@@ -8,9 +8,9 @@ generator_class: DefaultLuceneDocumentGenerator
 index_threads: 1
 index_options: -storePositions -storeDocvectors -storeRaw
 index_stats:
-  documents: 528036
```

Comment:
I think there's a bug here?

Reply:
Right, accidentally changed the old yaml when creating the new ones. I am fixing it.